### ISSUE TITLE
Features should be panned into view on focus

### ISSUE THREAD

**Issue by Malvoz** [Maps4HTML/Web-Map-Custom-Element#482]
Features should be panned into view on focus (similar to how popups are panned into view when opened). Otherwise a failure of WCAG 2.1 [SC 2.4.7 Focus Visible](https://www.w3.org/TR/WCAG21/#focus-visible).

**Comment by Malvoz** [comment]
Features outside the view are no longer treated as relevant for tabbing and thus not keyboard focusable (per https://github.com/Maps4HTML/Web-Map-Custom-Element/pull/596). And I believe there's currently no way for authors to programmatically focus a feature, so this is no longer needed AFAICT.

**Comment by Malvoz** [comment]
Closing for now. 🙂
